### PR TITLE
WFLY-14305 Some mistakes in Developer Guide

### PR DESCRIPTION
**Pull request by RanabirChakraborty**
Issue: https://issues.redhat.com/browse/WFLY-14305
Resolving Some mistakes in Developer Guide

### CODE REVIEW

**Comment by RanabirChakraborty** [comment]
@bstansberry if you can review this :)

File: docs/src/main/asciidoc/_developer-guide/Application_Client_Reference.adoc [hunk 1]
```diff
@@ -17,9 +17,9 @@ one].
 
 [IMPORTANT]
 
-Note that the application client is different to the EJB client
-libraries, it is perfectly possible to write client application that do
-not use the application client, but instead use the jboss-ejb-client
+Note that the application client is different from the EJB client
+libraries, it is perfectly possible to write client application that does
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/Application_Client_Reference.adoc:21]: s/write client/write a client/g

File: docs/src/main/asciidoc/_developer-guide/Application_Client_Reference.adoc [hunk 2]
```diff
@@ -17,9 +17,9 @@ one].
...
+not uses the application client, but instead use the jboss-ejb-client
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/Application_Client_Reference.adoc:22]: s/uses the/use the/g
s/instead use/instead uses/g

File: docs/src/main/asciidoc/_developer-guide/Implicit_module_dependencies_for_deployments.adoc [hunk 3]
```diff
@@ -16,7 +16,7 @@ Consider an application deployment which contains EJBs. EJBs typically
 need access to classes from the javax.ejb.* package and other Java EE
 API packages. The jars containing these packages are already shipped in
 WildFly and are available as "modules". The module which contains the
-javax.ejb.* classes has a specific name and so does the module which
+javax.ejb.* classes have a specific name and so does the module which
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/Implicit_module_dependencies_for_deployments.adoc:19]: "has" was correct as the subject is 'module' not 'classes'

File: docs/src/main/asciidoc/_developer-guide/JTA_Reference.adoc [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [[JTA_Reference]]
 = JTA Reference
 
-WildFly uses http://narayana.io[Narayana] as the implementation on of the JTA specification.
+WildFly uses http://narayana.io[Narayana] as the implementation on the JTA specification.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/JTA_Reference.adoc:4]: 'on' should be removed, and 'of' kept.

File: docs/src/main/asciidoc/_developer-guide/JTA_Reference.adoc [hunk 5]
```diff
@@ -169,7 +169,7 @@ NOTE: you can define the same behaviour when you use the `ejb-jar.xml`
       descriptor instead of the annotations
 
 The WildFly provides specific annotation `org.jboss.ejb3.annotation.TransactionTimeout`
-which gives you chance to change transaction timeout for particular bean/method.
+which gives you the chance to change transaction timeout for particular bean/method.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/JTA_Reference.adoc:172]: "which gives you the chance to change the transaction timeout for a particular bean/method"

File: docs/src/main/asciidoc/_developer-guide/Migrate_Seam_2_Booking_EAR_Binaries_-_Step_by_Step.adoc [hunk 6]
```diff
@@ -687,7 +687,7 @@ follow the new rules.
 ==== How to resolve it:
 
 To debug this, look earlier in the server log trace to what JNDI binding
-were used. Looking at the server log we see this:
+wes used. Looking at the server log we see this:
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/Migrate_Seam_2_Booking_EAR_Binaries_-_Step_by_Step.adoc:690]: s/wes/was

File: docs/src/main/asciidoc/_developer-guide/Migrate_Seam_2_Booking_EAR_Binaries_-_Step_by_Step.adoc [hunk 7]
```diff
@@ -838,7 +838,7 @@ login with demo/demo. You should the Booking welcome page.
 
 Although it would be much more efficient to determine dependencies in
 advance and add the implicit dependencies in one step, this exercise
-shows how problems appear in the log and provides some information on
+shows how problems appear in the log and provide some information on
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/Migrate_Seam_2_Booking_EAR_Binaries_-_Step_by_Step.adoc:841]: 'provides' was correct

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks @RanabirChakraborty

**Comment by RanabirChakraborty** [comment]
@bstansberry Thanks for the review :) changed it accordingly.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**